src/bin/vip-import-sql.js

```diff
@@ -204,15 +207,31 @@ command( {
 		console.log( `  importing: ${ chalk.blueBright( fileName ) }` );
 		console.log( `         to: ${ chalk.cyan( domain ) }` );
 		console.log( `       site: ${ app.name } (${ formatEnvironment( opts.env.type ) })` );
-		searchReplace ? '' : console.log();
+		if ( searchReplace && searchReplace.length ) {
+			const searchAndReplaceParams = searchReplace.split( ',' );
+			console.log(
+				`        s-r: ${ chalk.blue( searchAndReplaceParams[ 0 ] ) } -> ${ chalk.blue(
+					searchAndReplaceParams[ 1 ]
+				) }`
+			);
+		}
+		console.log();
+
+		// NO `console.log` after this point! It will break the progress printing.
 
 		let fileNameToUpload = fileName;
 
+		const progressTracker = new ProgressTracker( SQL_IMPORT_PREFLIGHT_PROGRESS_STEPS );
+		progressTracker.prefix = `=============================================================
+Processing your sql import for env ID: ${ env.id }, app ID: ${ env.appId }:\n`;
```

Comment:
Same thing as above- I think we should add these details to the summary printed initially when you start the command, or to the summary after the confirmation prompt so it looks cleaner.

(We can add that in a separate PR)

```suggestion
```